packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -61,6 +62,30 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 	);
 };
 
+interface TrackDesignViewProps {
+	categorization?: string | null;
```

Comment:
Small nitpick: `selectedCategory` / `category`? As `categorization` already has different shape in the caller code.